src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/GjennoptaBehandling/GjennoptaBehandling.tsx

```diff
@@ -13,15 +13,15 @@ interface IProps {
 }
 
 const GjennoptaBehandling: React.FC<IProps> = ({ behandling, onListElementClick }) => {
+    const lukkModalOgHentBehandling = () => {
```

Comment:
Legg denne under linjen der `settVisModal` opprettes slik at vi ikke refererer til funksjonen før den er opprettet :)

Reply:
ahh, jeg er for rask 🙈

Reply:
endret [ae6561e](https://github.com/navikt/familie-tilbake-frontend/pull/1744/commits/ae6561e4ccd81b346269cea7e3ae527c619e0671)